gem/migrations/0031_add_view_yourwords_entry_to_groups.py

```diff
@@ -0,0 +1,35 @@
+# flake8: noqa: E128
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-24 12:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.core.management.sql import emit_post_migrate_signal
+
+
+def add_view_yourwords_entry_to_groups(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+
+    emit_post_migrate_signal(2, False, db_alias)
+
+    Group = apps.get_model('auth.Group')
+    Permission = apps.get_model('auth.Permission')
+
+    view_yourwords_entry = Permission.objects.get(codename='can_view_entry')
+    group_names = ('product_admin', 'data_admin', 'data_viewer',
+                   'content_admin', 'content_admin')
+    for group_name in group_names:
+        group = Group.objects.get(name=group_name).first()
+        group.permissions.add(view_yourwords_entry)
+
```

Comment:
I'm confused by this method. How does we know that it the `can_view_entry` permission only exists for the YourWords model?

Reply:
@KaitCrawford you are right, not that I'm thinking about it, it is confusing, so the full permission is `yourwords | YourWords Competition Entry | Can View Entry` but I think I should have added the code name as `can_view_yourwords_entry`.